### DISCUSSION TITLE
Problem with search results

### DISCUSSION THREAD

**Discussion by XiuNgo** [MrGreensWorkshop/MrGreen-JekyllTheme#123] [Problems]
Clicking a link in the search results sometimes does not go to the article page but hides the search bar. I checked the css z-index: 999 but still didn't fix that.
Has anyone ever had such a thing?

**Comment by MrGreensWorkshop** [comment]
I think it was fixed in #70 , What OS and browser are you using for test?

**Reply by XiuNgo** [reply]
I'm using Win10. Both Microsoft Edge and Chrome have this. problem.

**Reply by MrGreensWorkshop** [reply]
This theme is not not tested in "Microsoft Edge" as I mentioned in [readme](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme#readme). For Chrome, after #70 I don't have any issue. (I've tested in different window sizes)

**Reply by XiuNgo** [reply]
My display resolution is 1366x768

**Reply by XiuNgo** [reply]
You can see it in: https://youtu.be/H38hUVdjyLw

**Reply by MrGreensWorkshop** [reply]
I watched the video. I opened my Chrome (macOS) and set the resolution to 1366x768 and I tested it again. It's not happening on my environment.

**Comment by Esanpons** [comment]
Hi, I am also having the same problem, and the truth is that I have taken the repository of the latest version and I don't know why it happens.

Do you have any solution?

**Reply by MrGreensWorkshop** [reply]
Hi, does it happens on your environment with the demo page below?

https://jekyll-theme-mrgreen-demo.mrgreensworkshop.com/

**Comment by incognito-developer** [comment]
Hi, I'm also having same problem, same as @XiuNgo 's youtube link. some pages are working, but some pages are not working on search page. I find the value with devtools(F12), It has correct href link. But only mouse clicks one blog's search page are not working well.

**Reply by incognito-developer** [reply]
in [this demo](https://jekyll-theme-mrgreen-demo.mrgreensworkshop.com/), I used "hello" keyword to search posts. only "projects" search box are working when I clicked.

But, when I reboot the PC, It also doesn't work like others.
